admin-tools.Dockerfile

```diff
@@ -10,15 +10,15 @@ WORKDIR /home/builder
 
 # cache Temporal packages as a docker layer
 COPY ./temporal/go.mod ./temporal/go.sum ./temporal/
-RUN (cd ./temporal && go mod download all)
+RUN --mount=type=cache,target=/root/.cache/go-build (cd ./temporal && go mod download all)
```

Comment:
Will this work locally? I want to be able to build server image locally.

Reply:
Yes, it does. I just confirmed and rebased on my other work